apps/estuary/lib/estuary/event_table.ex

```diff
@@ -9,6 +9,13 @@ defmodule Estuary.EventTable do
     |> Prestige.prefetch()
   end
 
+  def insert_event(author, create_ts, data, type) do
```

Comment:
Is it possible to minimize the no of paramaters, in the function?

Reply:
agreed. I'll pass it an event.